index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio:Pritikant Dhal</title>
+  <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+</head>
+<body>
+  <!--Header Section-->
+  <header class="flex">
+    <img src="images/icons/image_geometry_1.svg" alt="" />
```

Comment:
As stated by the previous reviewer, please make this image a background image instead of using the `<img>` tag. The reason for this is that the image in the design is decorative and should not be in the flow of the page. Also, styling it as a background image will increase your websites performance